index.html

```diff
@@ -73,6 +79,272 @@ <h1>I’m Sebastian. Glad to see you!</h1>
+                  </li>
+                </ul>
+              </div>
+
+              <button class="see-proj-btn-itself" type="button">
+                See Project
+              </button>
+            </div>
+          </div>
+
+          <div class="grid-box">
+            <img src="./styling-features-pictures/work-port-4.png" alt="" />
+
+            <div class="content-box-works">
+              <div class="title-grid-box">
+                <h2>Multi-Post Stories</h2>
+              </div>
+              <div class="canopy-row">
+                <h3>CANOPY</h3>
+                <ul>
+                  <li>• Back End Dev •</li>
+                  <li>2015</li>
+                </ul>
+              </div>
+              <p class="para-testi">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <div class="row-buttons-languages">
+                <ul class="ul-buttons-lg">
+                  <li class="li-buttons-lg">
+                    <button class="buttons-lg" type="button">html</button>
+                  </li>
+                  <li class="li-buttons-lg">
+                    <button class="buttons-lg" type="button">css</button>
+                  </li>
+                  <li class="li-buttons-lg">
+                    <button class="buttons-lg" type="button">javascript</button>
+                  </li>
+                </ul>
+              </div>
+
+              <button class="see-proj-btn-itself" type="button">
+                See Project
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- /* --- TESTIMONIALS --- */ -->
+
+    <!-- /* --- ABOUT MYSELF --- */ -->
+    <section class="about-myself headline">
+      <div class="intro-content intro">
+        <h1 class="title-about-myself">About Myself</h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+
+        <div class="connect-div">
+          <a href="#" class="connect-a">
+            <h3>LET'S CONNECT</h3>
+          </a>
+        </div>
+
+        <div class="social-icons about-ms">
+          <ul>
+            <li>
+              <a href="#"><i class="fa fa-instagram"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa fa-twitter"></i></a>
+            </li>
+            <li>
+              <a href="#"> <i class="fa fa-github"></i> </a>
+            </li>
+            <li>
+              <a href="#"><i class="fa fa-linkedin"></i></a>
+            </li>
+          </ul>
+        </div>
+
+        <button class="see-proj-btn-itself resume-btn" type="button">
```

Comment:
- [ ] All buttons on this project should respect and follow the button styling: Active, Hover, and Pressed (see design below):
![image](https://user-images.githubusercontent.com/71874425/159019666-25fa07d3-825f-49b0-a2f3-3999cc5d9a78.png)